deploy/foundation/hub/resources/crds/cluster.open-cluster-management.io_mirroredmanagedclusters.yaml

```diff
@@ -1,208 +1,172 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
I think we can delete this crd also, no one is using it.

Reply:
deleted